test/integration.js

```diff
@@ -0,0 +1,146 @@
+
+    it('should not have any broken links', function(done) {
+        const options = {
+            // https://github.com/stevenvachon/broken-link-checker
+            excludedKeywords: ['/2015', 'livereload'],
+            filterLevel: '3',
+        };
+
+        const errors = {};
+
+        const siteChecker = new blc.SiteChecker(options, {
+            link(result) {
+                if (result.broken) {
+                    errors[result.base.resolved] = errors[result.base.resolved] || [];
+                    errors[result.base.resolved].push(
+                        `${result.url.resolved} (${blc[result.brokenReason]})`
+                    );
+                }
+            },
+            site() {
+                if (Object.keys(errors).length > 0) {
+                    const msg = `Site has broken links: \n ${JSON.stringify(errors)}`;
+                    return done(new Error(msg));
+                }
+                return done();
+            },
+        });
+
+        siteChecker.enqueue(url, url);
+    });
+});
+
+test.describe('The landing page...', function() {
+    this.timeout(timeOut);
+    const url = baseURL;
+    const expectedTitle = 'View Source Conference 2016';
+
+    test.it(`should be called "${expectedTitle}"`, function(done) {
+        const driver = new webdriver.Builder()
+            .forBrowser('firefox')
+            .build();
+
+        driver.get(url).then(function() {
+            driver.getTitle().then(function(title) {
+                assert.equal(title, expectedTitle)
+            });
+            done();
+        });
+    });
+
+    test.it('should have a footer & footer should have a link', function(done) {
+        const selector = 'footer ul .footer_social_twitter a';
+        const expectedText = '@ViewSourceConf';
+
+        const driver = new webdriver.Builder()
+            .forBrowser('firefox')
+            .build();
+
+        driver.get(url).then(function() {
+            driver.findElement(webdriver.By.css(selector)).then(function(link) {
+                link.getText().then(function(text) {
+                    assert.equal(text.toLowerCase(), expectedText.toLowerCase());
+                });
+            });
+            done();
+        });
+    });
+});
+
+test.describe('The conference pages...', function() {
+    this.timeout(timeOut);
+
+    ['seattle-2016', 'berlin-2016'].forEach(function(conference) {
+        const url = `${baseURL}/${conference}`;
+
+        test.it(`should have a ${conference} register link`, function(done) {
+            const registerSelector = '.hero a[href*="ti.to"]';
+            const expectedText = 'Register';
+
+            const driver = new webdriver.Builder()
+                .forBrowser('firefox')
+                .build();
+
+            driver.get(url).then(function() {
+                driver.findElement(webdriver.By.css(registerSelector)).then(function(link) {
+                    link.getText().then(function(text) {
```

Comment:
Again, I would assert that the element is displayed here rather than the link text. Also, would it be worth clicking the link and asserting the registration pages load?